src/Mod/CAM/Roadmap/Epics/ImproveCoreDressups.md

```diff
@@ -0,0 +1,30 @@
+# STATUS:
+
+Active
+
+# Why it is a priority
+
+Some core FreeCAD functionalities are implemented in dressups, but these
+dressups are too hard for users to discover and configure. Where dressups
+provide core functionality, they should be simple and accessible to use.  They
+should also provide complete versions of this functionality.
+
+At time of epic creation, dressups are mentioned as "core essentials" in
+multiple of the "functionality" sections on the main roadmap README:
+
+- Operation configuration: entry/exit strategies are partly completed by
+  the lead in/out and ramp dressups
+- Operation configuration: holding tags are completed by the tag dressup
+
+# Scope
+
+This epic is about improving priority features that are provided by dressups.
+
+| In                                                                                     | Out |
+| ---                                                                                    | --- |
+| Computational performance improvements to the ramp dressup, so users never have to wait a long time for the dressup to run | Add features to the dogbone dressup, which provides niche functionality, not core functionality |
+| Implement a task panel UI for the ramp dressup                                         |     |
+| Make it possible to add and configure core dressups from within the operation UI panel |     |
+| Add new entrance/exit strategies to the lead in/out dressup                            |     |
+
```

Comment:
Currently the Dogbone dressup has a task panel but doesn't show it on creation.  This is weird and different from all other dressups.  Let's normalize it. 

Out of scope:  Improvements to the Array Dressup.   We need to do something about Array-like functions but neither the array operation or the current array dressup is the right way to go.  So let's address it in another epic.

Reply:
Ramp Entry has no task panel.